CHANGELOG.rst

```diff
@@ -1,10 +1,57 @@
 Change log
 ==========
 
-1.1.0 [unreleased]
-------------------
+Version 1.1.0 [2024-08-19]
+--------------------------
+
+Features
+~~~~~~~~
+
+- Added option to force devices to register using their MAC address
+  instead of their hostname.
+- Added support for `hotplug events
+  <https://openwisp.io/docs/dev/openwrt-config-agent/user/hotplug-events.html>`_.
+- Added error reporting functionality, which sends error logs to the
+  controller if the agent fails to apply a configuration update.
+- Implemented a retry mechanism with random backoff for essential
+  operations, including retrieving checksums and downloading
+  configurations from the controller, as well as reporting error statuses.
+
+Changes
+~~~~~~~
+
+- **DEPRECATION WARNING:**
+- Updated logging to capture errors from ``openwisp-update-config`` in
+  ``logd``.
+- Changed the default ``bootup_delay`` to ``10`` seconds.
+- Changed the auto-naming of ``network.device`` objects to make it
+  consistent with OpenWISP.
+
+Deprecations
+++++++++++++
+
+- Deprecated ``/usr/sbin/openwisp_config`` in favor of
+  ``/usr/sbin/openwisp-config`` to maintain consistency with other files
+  in the package. The path ``/usr/sbin/openwisp_config`` is now a symbolic
+  link to ``/usr/sbin/openwisp-config``.
```

Comment:
It's now a backward incompatible change due to https://github.com/openwisp/openwisp-config/pull/217

Reply:
Renaming of the `init.d` script is backward incompatible. This script is still present. 

https://github.com/openwisp/openwisp-config/pull/217/files#r1775225457